### PR TITLE
🌱 show full trace in CI for shell scripts

### PR DESCRIPTION
**Pull request by ykakarap**
**What this PR does / why we need it**:

With https://github.com/kubernetes-sigs/cluster-api/pull/7462 merged I think it is better to still keep the full trace at least in the CI.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

**Comment by sbueringer** [comment]
As far as I can tell nothing changed through #7462 

Logs before the merge:
* 590 lines https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_cluster-api/7462/pull-cluster-api-verify-main/1587442643602771968

Logs after merge on another PR:
* 590 lines https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_cluster-api/7268/pull-cluster-api-verify-main/1588338888244465664

Logs on this current PR:
* 14547 lines https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_cluster-api/7493/pull-cluster-api-verify-main/1588355221178814464

Or am I missing something?

**Comment by killianmuldoon** [comment]
I'm -1 to include this by default in our scripts. I think it's a good idea to have the option there for debugging, but on-by-default - like @sbueringer says - makes the output hard to parse.

**Comment by sbueringer** [comment]
@ykakarap Given comments above, do you think we can close this PR?

**Comment by ykakarap** [comment]
Yup agreed. Don't need this if it makes the logs harder to read. 

/close
